wasi-threads.abi.md

```diff
@@ -0,0 +1,34 @@
+# Types
+
+## <a href="#thread_id" name="thread_id"></a> `thread-id`: `u32`
+
+  Unique thread identifier.
+
+Size: 4, Alignment: 4
+
+## <a href="#errno" name="errno"></a> `errno`: enum
+
+  Error codes returned by the `thread-spawn` function.
+
+Size: 1, Alignment: 1
```

Comment:
isn't wasi errno 16-bit?

Reply:
This isn't the same as wasi's errno. Here we only define a subset of errors that can be returned by the function